vue/components/ui/molecules/search-persistent-local-storage/search-persistent-local-storage.vue

```diff
@@ -0,0 +1,94 @@
+<template>
+    <div class="search-persistent-local-storage">
+        <global />
+        <search-persistent
+            v-bind:value.sync="valueData"
+            v-bind:filters.sync="filtersData"
+            v-bind:placeholder="placeholder"
+            v-bind:icon-visible="iconVisible"
+            v-bind:clear-visible="clearVisible"
+            v-bind:variant="variant"
+            v-bind:width="width"
+            v-bind:loading="loading"
+            v-on:update:filters="onUpdateFilters"
+        />
+    </div>
+</template>
+
+<script>
+export const SearchPersistentLocalStorage = {
+    name: "search-persistent-local-storage",
+    props: {
+        variant: {
+            type: String,
+            default: null
+        },
+        value: {
+            type: String,
+            default: null
+        },
+        filters: {
+            type: Array,
+            default: () => []
+        },
+        placeholder: {
+            type: String,
+            default: "Search"
+        },
+        iconVisible: {
+            type: Boolean,
+            default: true
+        },
+        clearVisible: {
+            type: Boolean,
+            default: true
+        },
+        width: {
+            type: Number,
+            default: null
+        },
+        loading: {
+            type: Boolean,
+            default: false
+        }
+    },
+    data: function() {
+        return {
+            valueData: this.value,
+            filtersData: this.filters
+        };
+    },
+    watch: {
+        valueData(value) {
+            this.$emit("update:value", this.valueData);
+        },
+        filtersData(value) {
+            this.$emit("update:filters", this.filtersData);
+        }
+    },
+    computed: {
+        filtersKey() {
+            return this.$root.account ? `filters${this.$root.account._id}` : "filters";

```

Comment:
Is `this.$root.account` reactive? Please use the debugger to find out if it's an `Observable`. If it's not then `filtersKey` will need to be a method instead. Remember the `alert` not getting reset bug? It was caused by a similar problem (in that case, the `$refs` not being reactive).

Reply:
I checked, it is reactive.
Atleast in my knowledge, if it has the `__ob__` property it is reactive.

Reply:
Confirmed: https://github.com/ripe-tech/ripe-pulse/blob/2ce3ade6e91c18aa81b623f15a8db0e192f031c8/js/main.js#L207